todo/tests/unit/models/test_task.py

```
@@ -0,0 +1,47 @@
+from typing import List
```

Ruff (0.8.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)